tests/test_spark.py

```diff
@@ -105,6 +105,18 @@ def test_sparkPi(service_name=utils.SPARK_SERVICE_NAME):
         args=["--class org.apache.spark.examples.SparkPi"])
 
 
+@pytest.mark.sanity
+def test_spark_with_multi_configs(service_name=utils.SPARK_SERVICE_NAME):
+    utils.run_tests(
+        app_url=utils.dcos_test_jar_url(),
+        app_args="",
+        expected_output="spark.executor.extraJavaOptions,-XX:+PrintGCDetails -XX:+PrintGCTimeStamps -Dparam3=val3",
+        service_name=service_name,
+        args=["--conf spark.driver.extraJavaOptions='-XX:+PrintGCDetails -XX:+PrintGCTimeStamps -Dparam3=val3'",
```

Comment:
Could you add a few special characters here as well? Maybe a `-Dparam3=\"val3 val4\"` like in the unit test?